### ISSUE TITLE
How can I unpack FR Validation Set named lfw.bin or *.bin?

### ISSUE THREAD

**Issue by YoungjaeDev** [eeric/Face_recognition_cnn#7]
Hello 
CASIA-WebFace was able to unpack well after seeing the document(README) you uploaded, what can I do with the rest of the data (Validset)? 
And when I train FR, do I have to load all the datasets into Dataloader while unpacked?
Thank You

**Comment by eeric** [comment]
did not need unpack *.bin, it was a kind of dataset format, it was fit for insightface project.
